toffy/fov_watcher.py

```diff
@@ -62,17 +63,24 @@ def check_run_condition(self, path: str, check_interval: int = 10) -> Tuple[bool
                 whether or not both json and bin files exist, as well as the name of the point
         """
 
-        if not os.path.exists(path):
-            raise FileNotFoundError(f"{path} doesn't exist but was recently created. "
```

Comment:
Raising an error, but not returning anything, seems to stall the watcher. It doesn't actually exit from monitoring, but it won't process the subsequent FOVs that get generated. Maybe we should change the errors to warnings? For example the one at line 105?